src/libstd/process.rs

```diff
@@ -343,6 +359,23 @@ impl Command {
 
     /// Add an argument to pass to the program.
     ///
+    /// Only one argument can be passed per use.
+    /// So instead of:
+    ///
+    /// ```
+    /// .arg("-C /path/to/repo")
+    /// ```
+    /// usage would be:
```

Comment:
an extra `///` above this would be good, so there's space below just like the space above